doc/developer/index.md

```diff
@@ -26,7 +26,6 @@ Tool Development
 
 User Interface development
 --------------------------
-* [User Interface development](interface)
```

Comment:
Remove this change please.  Looks like it was an accidental deletion.

Reply:
Added back.